.github/workflows/container.yml

```diff
@@ -0,0 +1,57 @@
+name: Create and publish a container image
+
+on:
+  push:
+    # Publish main branch as `beta` image
+    branches:
+      - main
+      - master
+
+    # Publish tags as versioned release and `latest` image
+    tags:
+      - '*'
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
I think we should either:

- update the documentation, since ATM it assumes we are publishing the image on the Docker Hub registry
- also publish to Docker Hub, because AFAIK there are people already using that on their builds

Reply:
Which documentation you are referring? I didn't see any. I updated reference docker-compose files to use new GHCR.

Reply:
> Which documentation you are referring? I didn't see any.

That is the point, we need to document where the image is published. The commit with the pudated docker-compose.yml seems to be missing.

Reply:
True, I had forgot to push docker-compose updates. However I cannot see any references to Docker Hub here, so I don't understand what you expect to be changed now? Naturally at some point obsolete image from Docker Hub can be cleaned, but also we need to make new releases of Karapace. Now Docker Hub contains only one very old image, but which is also AFAIK latest versioned release of Karapace...

Reply:
> so I don't understand what you expect to be changed now? 

I guess you're saying that because GHCR won't start with an image of a release it should not be documented?